lib/core/src/display/camera/camera2d.rs

```diff
@@ -95,6 +95,12 @@ impl Default for Clipping {
 // === Camera2dData ===
 // ====================
 
+/// Function used to return the updated screen width, height and `Camera2d`'s `DisplayObjectData`.
+pub trait ScreenUpdateFn = FnMut(f32,f32,DisplayObjectData) + 'static;
```

Comment:
I don't expect function which updates screen dimensions to pass a `DisplayObjectData`. It is some internal object, it is a very strange design. If we tell camera to have a callback after screen update, it sohuld just call the callback. It can pass new scren dimension, but nothing more.